client/aliases.go

```diff
@@ -40,31 +41,21 @@ func (client *Client) performAliasAction(sa *aliasAction) (changeID string, err
 	return client.doAsync("POST", "/v2/aliases", nil, nil, bytes.NewReader(b))
 }
 
-// Alias enables the provided aliases for the snap with snapName.
-func (client *Client) Alias(snapName string, aliases []string) (changeID string, err error) {
+// Alias sets up a manual alias from alias to app in snapName.
+func (client *Client) Alias(snapName, app, alias string) (changeID string, err error) {
 	return client.performAliasAction(&aliasAction{
-		Action:  "alias",
-		Snap:    snapName,
-		Aliases: aliases,
+		Action: "alias",
+		Snap:   snapName,
+		Target: app,
+		Alias:  alias,
 	})
 }
 
-// Unalias disables explicitly the provided aliases for the snap with snapName.
-func (client *Client) Unalias(snapName string, aliases []string) (changeID string, err error) {
+// Unalias tears down a manual alias or disables all aliases of a snap (removing all manual ones)
```

Comment:
I think if separate them internally we should separate the CLI as well, given alias is only for manual aliases, seems unalias should be for that.

Then how do we want to call "snap unalias SNAP" then?